Makefile

```diff
@@ -1,5 +1,5 @@
 GO ?= go
-EPOCH_TEST_COMMIT ?= 78aae688e2932f0cfc2a23e28ad30b58c6b8577f
+EPOCH_TEST_COMMIT ?= 1cc5a27
```

Comment:
I thought it most likely was, but just wanted to make sure.  Thanks for verifying.